Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -20,4 +20,25 @@ def determine_rpath_paths
     paths += homebrew_extra_library_paths
     paths.to_path_s
   end
+
+  def x11
+    xorg = Formula["linuxbrew/xorg/xorg"]
+    if xorg.installed?
+      xorg.recursive_dependencies.each do |d|
+        dep_formula = d.to_formula
+        append_path "PKG_CONFIG_LIBDIR", dep_formula.lib/"pkgconfig"
+        append_path "PKG_CONFIG_LIBDIR", dep_formula.share/"pkgconfig"
```

Comment:
> Note that it needs to say recursive_dependencies and not just runtime_dependencies; otherwise libxcb doesn't add libxmcp to the list and thus pkg-config blows up for it.

If `libxcb` requires the `libxmcp.pc` file, then should `libxcb` depend directly on `libxmcp`, rather than indirectly through a recursive dependency?